pkg/pillar/cmd/zedagent/handlemetrics.go

```diff
@@ -1287,3 +1307,10 @@ func protoEncodeNetworkInstanceMetricProto(status types.NetworkInstanceMetrics)
 
 	return metric
 }
+
+// pathToKey converts /path/to/dir to path-to-dir.
+func pathToKey(path string) string {
+	path = strings.TrimPrefix(path, "/")
+	key := strings.ReplaceAll(path, "/", "-")
+	return key
+}
```

Comment:
You already have this in types package; better to export that function instead of duplicating it.

Reply:
Done.